src/app/shared/enum/enumUA/tech-admin/create-admin.ts

```diff
@@ -0,0 +1,5 @@
+export enum CreateAdminTitle {
+    ministryAdmin = 'АДМІНА МІНІСТЕРСТВА',
+    regionAdmin = 'АДМІНА РЕГІОНА',
+    territorialCommunityAdmin = 'АДМІНА ОТГ'
```

Comment:
I believe there is too much space, it should be 2 spaces

Reply:
fixed